addons/mail/static/src/components/discuss_sidebar_category/discuss_sidebar_category.scss

```diff
@@ -1,95 +1,9 @@
-// ------------------------------------------------------------------
-// Layout
-// ------------------------------------------------------------------
-
-.o_DiscussSidebarCategory_addingItemInput {
-    margin-left: $o-mail-discuss-sidebar-category-item-avatar-left-margin;
-    margin-right: $o-mail-discuss-sidebar-scrollbar-width;
-}
-
-.o_DiscussSidebarCategory_command {
-    margin-left: $o-mail-discuss-sidebar-category-item-margin;
-    margin-right:$o-mail-discuss-sidebar-category-item-margin;
-
-    &:first-child {
-        margin-left: 0;
-    }
-
-    &:last-child {
-        margin-right: 0;
-    }
-}
-
-.o_DiscussSidebarCategory_commands {
-    display: flex;
-}
-
-.o_DiscussSidebarCategory_header {
-    display: flex;
-    align-items: center;
-    margin: 5px 0;
-}
-
-.o_DiscussSidebarCategory_headerItem {
-    margin-left: $o-mail-discuss-sidebar-category-item-margin;
-    margin-right: $o-mail-discuss-sidebar-category-item-margin;
-
-    &:last-child {
-        margin-right: $o-mail-discuss-sidebar-scrollbar-width;
-    }
-}
-
-.o_DiscussSidebarCategory_title {
-    display: flex;
-    align-items: baseline;
-}
-
 .o_DiscussSidebarCategory_titleIcon {
     width: $o-mail-discuss-sidebar-category-title-icon-size;
     height: $o-mail-discuss-sidebar-category-title-icon-size;
 }
 
-
-// ------------------------------------------------------------------
-// style
-// ------------------------------------------------------------------
-
-.o_DiscussSidebarCategory_addingItemInput {
-    outline: none;
-}
-
-.o_DiscussSidebarCategory_command {
-    cursor: pointer;
-
-    &:not(:hover) {
-        color: gray('600');
-    }
-}
-
-.o_DiscussSidebarCategory_counter {
-    background-color: $o-brand-primary;
-    color: gray('300');
-}
-
 .o_DiscussSidebarCategory_headerItem {
     user-select: none;
```

Comment:
@karlosvalverde , you can keep it in this PR, but it should be addressed using the upcoming `user-select-none` BS5 utility class.
In another PR you should:
- introduce [user-select utility classes](https://getbootstrap.com/docs/5.1/utilities/interactions/#text-selection) in `bs5_utility_classes.scss`
- pass through  all the odoo code and apply the classes wherever is necessary (including this file)

Reply:
This will be addressed in the PR below:
https://github.com/odoo/odoo/pull/90931